Dockerfile

```diff
@@ -3,4 +3,10 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3-201
 RUN curl -sL -o /usr/bin/opa https://openpolicyagent.org/downloads/latest/opa_linux_amd64 && chmod 0755 /usr/bin/opa
 COPY policies /usr/share/opa/policies/
 
-ENTRYPOINT ["/usr/bin/opa", "run", "--server", "/usr/share/opa"]
+ENTRYPOINT ["/usr/bin/opa", \
```

Comment:
This will always enable TLS, right?
I'd rather make it a runtime decision based on environment variables, if you don't mind.

The `ENTRYPOINT` would then be a script that build the command line and calls `exec`.
If the `TLS_ENABLED`* variable is set to `true` (default: `false`), the script checks that the `TLS_CERT_FILE`*, `TLS_KEY_FILE`* and `TLS_CA_CERT_FILE`* are set and that the files exist. If all these conditions are met, it adds the TLS options to the command line.

This would allow to run without TLS in various context.
This would also allow to change the secret name and mount path in the deployment and service specs.

* example names

Reply:
I've made those changes, do they look ok?